src/client/components/GatewayErrorSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+	ErrorSummary,
+	ErrorSummaryProps,
+} from '@guardian/source-development-kitchen/react-components';
+
+import { asStructuredError, GatewayError } from '@/shared/model/Errors';
+import {
+	errorContextLastTypeSpacing,
+	errorContextSpacing,
+	errorMessageStyles,
+} from '@/client/styles/Shared';
+
+export interface GateWayErrorSummaryProps
+	extends Omit<ErrorSummaryProps, 'message' | 'cssOverrides'> {
+	shortRequestId?: string;
+	gatewayError?: GatewayError;
+}
+export default function GatewayErrorSummary(props: GateWayErrorSummaryProps) {
+	const structuredError = asStructuredError(props.gatewayError);
+	const fullContext =
+		props.shortRequestId && structuredError?.severity != 'BAU'
+			? [
+					props.context,
+					<p
+						css={[errorContextSpacing, errorContextLastTypeSpacing]}
+						key={'requestId'}
+					>
+						Request ID: {props.shortRequestId}
+					</p>,
+				]
+			: props.context;
+
+	const updatedProps = {
+		...props,
+		context: fullContext,
+		message: structuredError?.message,
+		cssOverrides: errorMessageStyles,
+	};
+	return <ErrorSummary {...updatedProps}> </ErrorSummary>;
```

Comment:
`updatedProps` has no type definition, its preferable to set them directly on the component

```suggestion
	return <ErrorSummary {...props} context={fullContext} message={structuredError?.message} cssOverrides={errorMessageStyles}> </ErrorSummary>;
```